### PR TITLE
Add date to front matter in new posts

### PR DESCRIPTION
**Pull request by xianny**
When making new posts, I'd like to auto-generate date/timestamp to front matter. Thoughts?

### CODE REVIEW

**Comment by lithium3141** [comment]
This is the very first feature I found myself wanting when first trying jekyll-compose. +1

**Comment by wrobbins** [comment]
Bump ?

**Comment by jandudulski** [comment]
@parkr any chances for this? :)

**Comment by pathawks** [comment]
LGTM

File: lib/jekyll/commands/post.rb [hunk 1]
```diff
@@ -53,6 +53,11 @@ def file_name
         def _date_stamp
           @params.date.strftime '%Y-%m-%d'
         end
+
+        def content
```
Comment (parkr) [inline lib/jekyll/commands/post.rb]: Subclasses should accept the same number of arguments as the superclass so this should be `def content(front_matter = {})`.

```ruby
def content(front_matter = {})
  front_matter.merge!({"date" => @params.data.strftime("%Y-%m-%d %H:%M %z")})
  super
end
```

That should work.

Comment (jandudulski) [reply lib/jekyll/commands/post.rb]: @xianny can you update this please :)?

**Pull request closed without merging**